server/game/cards/plots/00/commoncause.js

```diff
@@ -0,0 +1,32 @@
+const _ = require('underscore');
+
+const PlotCard = require('../../../plotcard.js');
+
+class CommonCause extends PlotCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: this,
+            effect: ability.effects.dynamicClaim(() => this.getNumberOfFactions())
+        });
+    }
+
+    getNumberOfFactions() {
+        let charactersInPlay = this.controller.filterCardsInPlay(card => card.getType() === 'character');
+        let factionsInPlay = [];
+
+        _.each(charactersInPlay, card => {
+            let factions = card.getFactions();
+            _.each(factions, faction => {
+                if(!factionsInPlay.includes(faction) && faction !== 'neutral') {
+                    factionsInPlay.push(faction);
```

Comment:
This whole things makes my teeth chatter but it'll work. 😬  The more functional way of expressing this isn't a lot better in this case.

You could potentially get rid of the neutral check by filtering for non-neutral characters in play.